sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.0.0-preview.6 (2022-06-07)
 
-### Changed
+### Other Changes
```

Comment:
we should not change the changelog entry for released version, it is meaningless.